edb/schema/rewrites.py

```diff
@@ -223,7 +233,9 @@ def get_dummy_expr_field_value(
         value: Any,
     ) -> Optional[s_expr.Expression]:
         if field.name == 'expr':
-            return s_expr.Expression(text='false')
+            pt = self.scls.get_ptr_target(schema)
+            text = f'<{pt.get_displayname(schema)}>{{}}'
```

Comment:
So this is just a cast of an empty set to the correct type?

Reply:
Yeah